controllers/managedocs_controller.go

```diff
@@ -44,17 +51,37 @@ type ManagedOCSReconciler struct {
 	Scheme *runtime.Scheme
 	ctx    context.Context
 
-	managedOCS *v1.ManagedOCS
+	managedOCS       *v1.ManagedOCS
+	ConfigSecretName string
```

Comment:
I think this is not a good name for the field. The field is not a general config nor is it the only configuration source.
As such I would prefer something more specific like `AddOnParamsSecretName`

Reply:
Done